FLEDGE.md

```diff
@@ -151,28 +150,35 @@ const myAuctionConfig = {
                         'https://www.another-buyer.com': 200,
                         '*': 150,
                         ...},
+  'componentAuctions': [
+    {'seller': 'https://www.some-other-ssp.com',
+      ...},
+    ...
+  ]
 };
 const auctionResultPromise = navigator.runAdAuction(myAuctionConfig);
 ```
 
 
 This will cause the browser to execute the appropriate bidding and auction logic inside a collection of dedicated worklets associated with the buyer and seller domains.  The `auctionSignals`, `sellerSignals`, and `perBuyerSignals` values will be passed as arguments to the appropriate functions that run inside those worklets — the `auctionSignals` are made available to everyone, while the other signals are given only to one party.
 
+In some cases, multiple SSPs may want to run their own auction and send the winner up to another auction. To facilitate these "component auctions", `componentAuctions` can optionally contain additional auction configurations for each seller's "component auction".  The winning bid of each of these "component auctions" will be passed to the top-most auction.  How bids are scored in this case is further described in [2.4 Scoring Bids in Component Auctions](#24-scoring-bids-in-component-auctions). The `AuctionConfig` of component auctions may not have their own `componentAuctions`.
```

Comment:
Different SSPs may charge different seller fees, and for the top-level auction to maximize publisher payout, winning bids from each component auction may need to be compared expressed in terms of publisher payout, net of those fees. Such fees and other business arrangements may determine how an impression cost paid by the winning buyer is allocated between a publisher, a component seller and, sometimes, a top-level seller.

Could a "component seller" somehow choose to specify a bid value to be submitted into the top-most auction that is derived – but may be different – from the bid value that its buyers submit, to account for fees and other business arrangements that may be in place between a publisher, a "component" seller and a top-level seller?

Reply:
I'll defer to @michaelkleber on this one.

Reply:
@sbelov makes a very good point here. I think the goal would be that a component auction can run business logic that determines final bid price (eg pub payout) in the component auction and represent that in the final auction.

Reply:
I've added an option bid parameter to the output of scoreAd for component auctions, which is also passed to the component seller's reportResult method as a new value (with bid being the original bid).